conans/util/windows.py

```diff
@@ -48,6 +50,16 @@ def path_shortener(path, short_paths):
         drive = os.path.splitdrive(path)[0]
         short_home = drive + "/.conan"
     mkdir(short_home)
+
+    # Workaround for short_home living in NTFS file systems. Give full control permission to current user to avoid
+    # access problems in cygwing/msys2 windows subsystems when using short_home folder
+    try:
+        cmd = r'cacls %s /E /G "%s\%s":F' % (short_home, os.environ['USERDOMAIN'], os.environ['USERNAME'])
+        subprocess.check_output(cmd, stderr=subprocess.STDOUT)  # Ignoring any returned output, make command quiet
```

Comment:
why not calling `os.system`? I don't get it. Is it to hide the output? Why?

Reply:
I want to do the command silent, the command can fail if short_home if non NTFS file system.

Reply:
Ah, ok, yeah, maybe an error trace is confusing.